config/locales/views.ja.yml

```
@@ -31,3 +31,4 @@ ja:
     new:
       title: "注文"
       purchase: "注文する"
+      delivery_on_hint: "3営業日後（営業日: 月-金）から14営業日までを指定してください"
```

細かい）
営業日をDRYにしたいなとおもった
```suggestion
      delivery_on_hint: "3〜14営業日までを指定してください（営業日: 月-金）"
```